Tutorial is out of date

If I run the setup as described I get the following error.

```
➜  jspecify-reference-checker git:(main) ./demo SimpleSample.java
SimpleSample.java:1: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.NullMarked;
                               ^
SimpleSample.java:2: error: package org.jspecify.annotations does not exist
import org.jspecify.annotations.Nullable;
                               ^
SimpleSample.java:4: error: cannot find symbol
@NullMarked
 ^
  symbol: class NullMarked
SimpleSample.java:6: error: cannot find symbol
  Object passThrough(@Nullable Object mightBeNull) {
                      ^
  symbol:   class Nullable
  location: class SimpleSample
SimpleSample.java:5: error: [nullness] NullSpecChecker did not run because of a previous error issued by javac
class SimpleSample {
^
5 errors
Note:
  =====
  The nullness checker is still early in development.
  It has many rough edges.
  For an introduction, see https://jspecify.dev/docs/user-guide
  =====
```

I'm aiming to try out the nullity annotations on a no-dep library i've made and confirm that i've annotated things correctly before publishing a new version.

--- a/build.gradle
+++ b/build.gradle
@@ -54,6 +54,7 @@ dependencies {
 
 // Assemble checker-framework when assembling the reference checker.
 assemble.dependsOn(gradle.includedBuild("checker-framework").task(":assemble"))
+assemble.dependsOn(gradle.includedBuild("jspecify").task(":assemble"))
 
 tasks.withType(JavaCompile).all {
     options.compilerArgs.add("-Xlint:all")

